libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -744,10 +744,8 @@ - (void)handlePostLogout
 
 - (void)handleUserSwitch:(SFUserAccount *)fromUser toUser:(SFUserAccount *)toUser
 {
-    // Close the passcode screen and reset passcode monitoring.
-    [SFSecurityLockout cancelPasscodeScreen];
-    [SFSecurityLockout stopActivityMonitoring];
-    [SFSecurityLockout removeTimer];
+    [SFSecurityLockout setupTimer];
```

Comment:
This is kind of incongruent: one delegate event has a helper method, the other does not, with no discernible reason why.  Should we normalize those code blocks?

Reply:
@khawkins Would it be better to change the interface of SalesforceSDKManagerFlow for the user switching to 
from 
```
- (void)handleUserSwitch:(nullable SFUserAccount *)fromUser toUser:(nullable SFUserAccount *)toUser
```
to 
```
```

Reply:
I can additionally normalize those blocks of code as well.

Reply:
That'd be ideal, to me.  The "flow" protocol is meant primarily for better ease of unit testing, so any additions there would better facilitate that process.

Reply:
Sorry, I mean your first suggestion would be ideal.